asf_heat_pump_suitability/getters/load_tree_input.py

```diff
@@ -0,0 +1,60 @@
+import geopandas as gpd
+import pandas as pd
+from typing import Optional, List
+from asf_heat_pump_suitability import config
+
+
+def load_gdf_os_openmap_local_layer(
+    layer: str, grid_squares: Optional[List[str]] = None, **kwargs
+) -> gpd.GeoDataFrame:
+    """
+    Load specified OS OpenMap Local layer. CRS British National Grid (27700).
+
+    Args:
+        layer (str): name of layer to load.
+        grid_squares (Optional[List[str]]): names of grid squares in OS mapping for regions of Great Britain to be loaded.
+        Default None to load whole GB.
+
```

Comment:
```suggestion
   **kwargs for geopandas file reader
```
Suggestion for additional documentation